addons/medical_status/functions/fnc_hasStableVitals.sqf

```diff
@@ -31,4 +31,13 @@ if (_bloodPressureL < 50 || {_bloodPressureH < 60}) exitWith { false };
 private _heartRate = GET_HEART_RATE(_unit);
 if (_heartRate < 40) exitWith { false };
 
+private _spo2 = GET_SPO2(_unit);
+if (_spo2 < 80) exitWith { false };
+
+private _blocked = GET_AIRWAY_BLOCKED(_unit);
+if (_blocked) exitWith { false };
+
+private _collapsed = GET_AIRWAY_COLLAPSED(_unit);
+if (_collapsed) exitWith { false };
+
```

Comment:
In real life you can become concious with either blocked or collapsed airways, I don't think that should be a stable vital to check.

If you have collapsed airways, but your oxygen saturation is high enough, you just get conscious and have your muscle reflexes back to move your tongue out of the way,
The same goes for blocked (vomit) you will feel it and just spit it out, ever drank something while lying on your back? Same procedure